Index: lclstrconsts.pas
===================================================================
--- lclstrconsts.pas	(revision 28643)
+++ lclstrconsts.pas	(working copy)
@@ -32,70 +32,70 @@
   SNoMDIForm = 'No MDI form present.';
 
   // message/input dialog buttons
-  rsMbYes          = '&Yes';
-  rsMbNo           = '&No';
-  rsMbOK           = '&OK';
-  rsMbCancel       = 'Cancel';
-  rsMbAbort        = 'Abort';
-  rsMbRetry        = '&Retry';
-  rsMbIgnore       = '&Ignore';
-  rsMbAll          = '&All';
-  rsMbNoToAll      = 'No to all';
-  rsMbYesToAll     = 'Yes to &All';
-  rsMbHelp         = '&Help';
-  rsMbClose        = '&Close';
-  rsmbOpen         = '&Open';
-  rsmbSave         = '&Save';
-  rsmbUnlock       = '&Unlock';
+  rsMbYes          = '&Sim';
+  rsMbNo           = '&Não';
+  rsMbOK           = '&Ok';
+  rsMbCancel       = 'Cancelar';
+  rsMbAbort        = 'Abortar';
+  rsMbRetry        = '&Repetir';
+  rsMbIgnore       = '&Ignorar';
+  rsMbAll          = '&Todos';
+  rsMbNoToAll      = 'Não para todos';
+  rsMbYesToAll     = 'Sim para t&odos';
+  rsMbHelp         = '&Ajuda';
+  rsMbClose        = '&Fechar';
+  rsmbOpen         = '&Abrir';
+  rsmbSave         = '&Salvar';
+  rsmbUnlock       = '&Desbloquear';
 
-  rsMtWarning      = 'Warning';
-  rsMtError        = 'Error';
-  rsMtInformation  = 'Information';
-  rsMtConfirmation = 'Confirmation';
-  rsMtAuthentication = 'Authentication';
+  rsMtWarning      = 'Aviso';
+  rsMtError        = 'Erro';
+  rsMtInformation  = 'Informação';
+  rsMtConfirmation = 'Confirmação';
+  rsMtAuthentication = 'Autenticação';
   rsMtCustom       = 'Custom';
 
   // file dialog
-  rsfdOpenFile           = 'Open existing file';
-  rsfdOverwriteFile      = 'Overwrite file ?';
-  rsfdFileAlreadyExists  = 'The file "%s" already exists. Overwrite ?';
-  rsfdPathMustExist      = 'Path must exist';
-  rsfdPathNoExist        = 'The path "%s" does not exist.';
-  rsfdFileMustExist      = 'File must exist';
-  rsfdDirectoryMustExist = 'Directory must exist';
-  rsfdFileNotExist       = 'The file "%s" does not exist.';
-  rsfdDirectoryNotExist  = 'The directory "%s" does not exist.';
+  rsfdOpenFile           = 'Abrir arquivo existente';
+  rsfdOverwriteFile      = 'Substituir arquivo?';
+  rsfdFileAlreadyExists  = 'O arquivo "%s" já existe. Substituir?';
+  rsfdPathMustExist      = 'Caminho deve existir';
+  rsfdPathNoExist        = 'O caminho "%s" não existe.';
+  rsfdFileMustExist      = 'Arquivo deve existir';
+  rsfdDirectoryMustExist = 'Diretório deve existir';
+  rsfdFileNotExist       = 'O arquivo "%s" não existe.';
+  rsfdDirectoryNotExist  = 'O diretó "%s" não existe.';
   rsFind = 'Find';
-  rsfdFileReadOnlyTitle  = 'File is not writable';
-  rsfdFileReadOnly       = 'The file "%s" is not writable.';
-  rsfdFileSaveAs         = 'Save file as';
-  rsAllFiles = 'All files (%s)|%s|%s';
-  rsfdSelectDirectory    = 'Select Directory';
-  rsDirectory            = '&Directory';
+  rsfdFileReadOnlyTitle  = 'O arquivo não é gravál';
+  rsfdFileReadOnly       = 'O arquivo "%s" não é gravável.';
+  rsfdFileSaveAs         = 'Salvar arquivo como';
+  rsAllFiles = 'Todos os arquivos (%s)|%s|%s';
+  rsfdSelectDirectory    = 'Selecionar diretório';
+  rsDirectory            = '&Diretório';
 
   // Select color dialog
-  rsSelectcolorTitle    = 'Select color';
+  rsSelectcolorTitle    = 'Selecionar cor';
    
   // Select font dialog
-  rsSelectFontTitle     = 'Select a font';
-  rsFindMore = 'Find more';
-  rsReplace = 'Replace';
-  rsReplaceAll = 'Replace all';
+  rsSelectFontTitle     = 'Selecionar uma fonte';
+  rsFindMore = 'Pesquisar mais';
+  rsReplace = 'Substituir';
+  rsReplaceAll = 'Substituir todos';
   
   // DBGrid
-  rsDeleteRecord = 'Delete record?';
+  rsDeleteRecord = 'Excluir registro?';
 
   // DBCtrls
-  rsFirstRecordHint = 'First';
-  rsPriorRecordHint = 'Prior';
-  rsNextRecordHint = 'Next';
-  rsLastRecordHint = 'Last';
-  rsInsertRecordHint = 'Insert';
-  rsDeleteRecordHint = 'Delete';
-  rsEditRecordHint = 'Edit';
-  rsPostRecordHint = 'Post';
-  rsCancelRecordHint = 'Cancel';
-  rsRefreshRecordsHint = 'Refresh';
+  rsFirstRecordHint = 'Primeiro';
+  rsPriorRecordHint = 'Anterior';
+  rsNextRecordHint = 'Próximo';
+  rsLastRecordHint = 'Último';
+  rsInsertRecordHint = 'Inserir';
+  rsDeleteRecordHint = 'Excluir';
+  rsEditRecordHint = 'Editar';
+  rsPostRecordHint = 'Salvar';
+  rsCancelRecordHint = 'Cancelar';
+  rsRefreshRecordsHint = 'Atualizar';
   
   // gtk interface
   rsWarningUnremovedPaintMessages = ' WARNING: There are %s unremoved LM_'
